config_examples/README.md

```diff
@@ -7,6 +7,12 @@ Dynatrace distribution of the OpenTelemetry Collector.
 > The examples in this directory are for documentation purposes only and are not considered stable. Examples
 > may change at any time and without notice.
 
+> [!CAUTION]
+> It is generally preferable to bind endpoints to localhost when all clients are local.
+> As of this release, that is also the default, but for convenience, our example 
```

Comment:
We could link to the release like this (not available yet but it will link correctly when the release is published).

```suggestion
> As of [v0.9.0](https://github.com/Dynatrace/dynatrace-otel-collector/releases/tag/v0.9.0), that is also the default, but for convenience, our example 
```

This way users immediately know which version this change was made in. :slightly_smiling_face: